assets/css/styles.css

```diff
@@ -0,0 +1,36 @@
+@import url('https://fonts.googleapis.com/css?family=Rubik:300,400&display=swap');
+
+body{ 
+    width: 100%; 
+    height: 100%; 
+    margin: 0px; 
+    padding: 0px; 
+    background: #E5E5E5;
+}
+
+*{
+    font-family: 'Rubik', sans-serif !important;
+    font-weight: 300 !important;
+}
+
+.form-container{
+    padding: 4em 3em;
```

Comment:
Nit: Can you use rem instead of em please? Every though there are lots of debates on the best to use between rem and em, I prefer rem since it is easier to use doesn’t cause weird bugs.